src/components/drag_and_drop/draggable.tsx

```diff
@@ -40,6 +40,11 @@ export interface EuiDraggableProps
    * Whether the `children` will provide and set up its own drag handle
    */
   customDragHandle?: boolean;
+  /**
+   * Whether the container is draggable and should have `role="group"` instead of `"button"`.
```

Comment:
One last minor documentation suggestion based on the new prop name and I think this is good to go!

```suggestion
   * Whether the container has interactive children and should have `role="group"` instead of `"button"`.
```